### PR TITLE
chore: add docs on error types for liveness

### PR DESCRIPTION
**Pull request by thaddmt**
#### Description of changes
- added a table for error states 
- https://liveness-amplify-cli-link.d2gpzrwq1v94nu.amplifyapp.com/react/connected-components/liveness#error-states

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/src/pages/[platform]/connected-components/liveness/react-props.ts [hunk 1]
```diff
@@ -88,3 +88,53 @@ export const FACE_LIVENESS_DETECTOR_CONFIG = [
...
+  {
+    name: `SERVER_ERROR`,
+    description:
+      'Indicates an error occurred from the Rekognition streaming API or in the onAnalysisComplete callback.',
+    type: `Error`,
+  },
+  {
+    name: `TIMEOUT`,
+    description:
+      'Indicates an timeout occurred when user attempted a liveness check.',
```
Comment (ioanabrooks) [inline docs/src/pages/[platform]/connected-components/liveness/react-props.ts:107]:
```suggestion
      'Indicates a timeout occurred when user attempted a liveness check.',
```

File: docs/src/pages/[platform]/connected-components/liveness/full-api.react.mdx [hunk 2]
```diff
@@ -22,6 +22,12 @@ Below is the full list of props that can be used with the `FaceLivenessDetectorC
 
 <ReactPropsTable props={FACE_LIVENESS_DETECTOR_CONFIG} />
 
+### Error States
+
+Below is the full list of error states that can be returned from the `onError` prop. You can also reference the [source code here](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-liveness/src/components/FaceLivenessDetector/service/types/liveness.ts#L102).
```
[the remark this answers is not in the document]
Comment (reesscot) [reply docs/src/pages/[platform]/connected-components/liveness/full-api.react.mdx:27]: +1, could we use the error types to a separate file?

Also, are we going to link to the default error messages as well for easier translation via displayText by customers?

**Comment by thaddmt** [comment]
Created separate pr for moving error files

File: docs/src/pages/[platform]/connected-components/liveness/full-api.react.mdx [hunk 3]
```diff
@@ -18,10 +18,16 @@ Below is the full list of props that can be used with the `FaceLivenessDetectorC
...
 ### FaceLivenessDetectorConfig
 
-Below is the full list of props that can be used with the `FaceLivenessDetectorConfig`. You can also reference the [source code here](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-liveness/src/components/FaceLivenessDetector/service/types/liveness.ts#L42).
+Below is the full list of props that can be used with the `FaceLivenessDetectorConfig`. You can also reference the [source code here](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-liveness/src/components/FaceLivenessDetector/service/types/error.ts). For the error text strings used for translation you can view the [display text file](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts).
 
 <ReactPropsTable props={FACE_LIVENESS_DETECTOR_CONFIG} />
 
+### Error States
+
+Below is the full list of error states that can be returned from the `onError` prop. You can also reference the [source code here](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-liveness/src/components/FaceLivenessDetector/service/types/liveness.ts#L102).
```
Comment (ioanabrooks) [inline docs/src/pages/[platform]/connected-components/liveness/full-api.react.mdx:27]: Should this reference the new errors.ts file?

Comment (thaddmt) [reply docs/src/pages/[platform]/connected-components/liveness/full-api.react.mdx:27]: Hm... I swear i changed this

Comment (thaddmt) [reply docs/src/pages/[platform]/connected-components/liveness/full-api.react.mdx]: Fixed!

**Review by reesscot — APPROVED** [review]

**Review by ioanabrooks — APPROVED** [review]

**Pull request merged**